lib/hanami/config.rb

```diff
@@ -104,7 +104,11 @@ class Config
     # @!attribute [rw] no_auto_register_paths
     #   Sets the paths to skip from container auto-registration.
     #
-    #   Defaults to `["entities"]`.
+    #   Paths are relative to the app root or the slice root.
```

Comment:
```suggestion
    #   Paths are relative to `/app` or a slice root.
```

I think this would be clearer and more correct?

"App root" I would consider to be the `Hanami.app.root`, which will be the root of the whole project, not the `/app` directory.